test/test_datapoints.py

```diff
@@ -113,6 +113,26 @@ def test_detach_wrapping():
     assert type(image_detached) is datapoints.Image
 
 
+def test_no_wrapping_exceptions_with_metadata():
+    # Sanity checks for the ops in _NO_WRAPPING_EXCEPTIONS and datapoints with metadata
```

Comment:
If you're wondering why I added this test, it's because I originally had changed the `_NO_WRAPPING_EXCEPTIONS` to return `output.as_subclass(cls)` instead of `cls.wrap_like(input, output)`, and I didn't get a proper error in the tests. The only tests that were failing here were `test_deepcopy()` which was failing on `print(bbox)`, which is a bit too "remote" of a test.